pkg/webui/console/components/webhook-form/index.js

```diff
@@ -360,19 +370,30 @@ export default class WebhookForm extends Component {
   }
 
   render() {
-    const { update, initialWebhookValue, webhookTemplate, healthStatusEnabled, error } = this.props
+    const {
+      update,
+      initialWebhookValue,
+      webhookTemplate,
+      healthStatusEnabled,
+      webhookRetryInterval,
+      isUnhealthyWebhook,
+      error,
+    } = this.props
+    let retryIntervalValue
+    let retryIntervalIntlUnit
+    if (webhookRetryInterval) {
+      retryIntervalValue = webhookRetryInterval.split(unitRegexp)[0]
+      const retryIntervalUnit = webhookRetryInterval.split(retryIntervalValue)[1]
+      retryIntervalIntlUnit = units.find(({ value }) => value === retryIntervalUnit).unit
```

Comment:
```suggestion
      // console/lib/regexp.js
      const duration = /^[0-9]+([a-z])$/
      
      retryIntervalUnit = webhookRetryInterval.match(durationRegExp)?[1]
      retryIntervalIntlUnit = units[retryIntervalUnit] // see above about reshaping the units variable
```
Haven't tested but this would be more concise.